recipes/rp2biosensor/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "rp2biosensor" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brsynth/rp2biosensor/archive/refs/tags/{{ version }}.tar.gz
```

Comment:
i'm not an official reviewer, but: if at all possible, following the general trend of a "canonical" upstream source release on, in this case, PyPI, would be preferable to use of a github source tarball. This helps keep automated tooling that relies on this namespace up-to-date.

Reply:
As part of that, you'd want to ensure the license file gets included in the distribution, by adding `include LICENSE.md` to `MANIFEST.in`